spec/models/user_spec.rb

```diff
@@ -10,4 +11,17 @@
     expect(@user.email).to match 'user@example.com'
   end
 
+  it "#role is :user by default" do
+    expect(@user.role).to eq "user"
+  end
+
+  it "#role can be set to :vip" do
+    @user.role = :vip
+    expect(@user.role).to eq "vip"
```

Comment:
This is a really weird Devise-ism, right? It's confusing to me that we assign symbols to the role but get back strings. I guess this is okay, though.